src/System.Management.Automation/security/SecuritySupport.cs

```diff
@@ -1140,7 +1085,7 @@ public void Resolve(SessionState sessionState, ResolutionPurpose purpose, out Er
             if (_pendingCertificate != null)
             {
                 ProcessResolvedCertificates(purpose,
-                    new List<X509Certificate2> { _pendingCertificate }, out error);
+                    new X509Certificate2Collection(_pendingCertificate), out error);
```

Comment:
Arguments should be on separate lines.
```suggestion
                    ProcessResolvedCertificates(
                        purpose,
                        new X509Certificate2Collection(_pendingCertificate),
                        out error);
```